src/core.c/Range.pm6

```diff
@@ -786,6 +798,9 @@ multi sub prefix:<^>(Int:D $max) {
     $range
 }
 multi sub prefix:<^>($max) { Range.new(0, $max.Numeric, :excludes-max) }
+multi sub prefix:<^>(Version:D $max) {
```

Comment:
Are you sure about the placement of this sub? Did you consider putting it in the same file as Version, to be consistent with the one above (`Range` as argument, placed in `Range.pm6`)

Reply:
Good catch.  Fixed with https://github.com/rakudo/rakudo/commit/8378794657